apis/fluent/atrium-api-fluent/src/jvmTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/SQLExceptionFeaturesSamples.kt

```diff
@@ -0,0 +1,80 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.expect
+import ch.tutteli.atrium.creating.Expect
+import errorCode
+import sqlState
+import java.sql.SQLException
+import kotlin.test.Test
+
+class SQLExceptionFeaturesSamples {
+
+    @Test
+    fun errorCodeFeature() {
+        val exception = SQLException("Test exception", "42000", 1001)
+
+        expect(exception)
+            .errorCode // subject is now of type Int
+            .toEqual(1001)
+
+        fails {
+            expect(exception)
+                .errorCode // subject is now of type Int
+                .toEqual(9999) // fails
+                .toBeGreaterThan(1000) // not evaluated/reported because `toEqual` already fails
+            // use `.errorCode { ... }` if you want all assertions evaluated
+        }
```

Comment:
I suggest you align the comments along the line of:
```suggestion
        fails {
            expect(exception)
                .errorCode              // subject is now of type Int
                .toEqual(9999)          // fails
                .toBeGreaterThan(1000)  // not evaluated/reported because `toEqual` already fails
            //                             use `.errorCode { ... }` if you want all assertions evaluated
        }
```